### PR TITLE
fix: load intl polyfill locale data

### PR DESCRIPTION
**Pull request by limpbrains**
### Description

Because we did not load the polyfill language files, some translations were missing

### Type of change

Bug fix

### Tests

No test

### Screenshot / Video

German before and after
 

### QA Notes

Make sure other languages that we support are showing correct dates on android

### CODE REVIEW

File: src/utils/i18n/index.ts [hunk 1]
```diff
@@ -58,7 +62,27 @@ i18nICU
...
+	.then(async () => {
+		// we need to load language related polyfill data
+		const lang = i18nICU.language;
+		try {
+			// @ts-ignore
+			if (NumberFormat.polyfilled) {
+				await numberFormatPolyfills[lang]?.();
+			}
+			// @ts-ignore
+			if (Intl.PluralRules.polyfilled) {
```
Comment (pwltr) [inline src/utils/i18n/index.ts:76]: Did you mean to set this somewhere?

File: src/utils/i18n/index.ts [hunk 2]
```diff
@@ -58,7 +62,27 @@ i18nICU
...
+				await pluralRulesPolyfills[lang]?.();
+			}
+			// @ts-ignore
+			if (Intl.RelativeTimeFormat.polyfilled) {
```
Comment (pwltr) [inline src/utils/i18n/index.ts:80]: Same here?

**Review by pwltr — CHANGES_REQUESTED** [review]

File: src/utils/i18n/index.ts (hunk 1 above)
Comment (limpbrains) [reply src/utils/i18n/index.ts:76]: This is set by polyfill library itself

Comment (pwltr) [reply src/utils/i18n/index.ts:76]: I see

**Review by pwltr — APPROVED** [review]

**Pull request merged**